cli/types/cypress.d.ts

```diff
@@ -5284,7 +5290,7 @@ declare namespace Cypress {
      * Useful to modify the window on a page transition.
      * @see https://on.cypress.io/catalog-of-events#App-Events
      */
-    (action: 'window:before:load', fn: (win: AUTWindow) => void): Cypress
+    (action: 'window:before:load', fn: (win: AUTWindow, eventHandlerWrap?: EventHandlerWrap | undefined) => void): Cypress
```

Comment:
I don't know if we want or need to update the public API for this. The `app:window:before:load` event is internal, so I think this can be accomplished by only changing its signature.